testing_and_setup/compass/ocean/hurricane/hurricane_wind_pressure/requirements.txt

```diff
@@ -0,0 +1,32 @@
+atomicwrites==1.2.1
```

Comment:
Need to double-check to make sure none of these are being deprecated.

Reply:
I'm going to remove this file, as we discussed.